### PR TITLE
type="username" is not valid html and breaks firefox autofill; changed to "text"

### PR DESCRIPTION
**Pull request by bricas**

### CODE REVIEW

File: public/css/login.css [hunk 1]
```diff
@@ -36,7 +36,7 @@ form
 	padding:6% 4%;
 }
 
-input[type="username"], input[type="password"]
+input[name="text"], input[type="password"]
```
Comment (daN4cat) [inline public/css/login.css]: Not sure it's name="text". Isn't type="text" and name="username"?

**Comment by daN4cat** [comment]
Good catch but css needs fixing.

File: public/css/login.css (hunk 1 above)
Comment (bricas) [reply public/css/login.css]: sigh. the mind i strange thing sometimes! :) In any case, this should now be fixed.

**Comment by daN4cat** [comment]
Merged and thanks for the contribution.

File: public/css/login.css (hunk 1 above)
Comment (daN4cat) [reply public/css/login.css]: That's why we review :-)

**Pull request merged**